go/v1beta1/storage/storage.go

```diff
@@ -95,6 +96,37 @@ func (p GrafeasStorageProvider) Provide(_ string, confi *config.StorageConfigura
 	return grafeasStorage, nil
 }
 
+// ProvideRW returns a storage which is configured based on the receiver's fields. The storage connects to different reader/writer. If no reader is provided, then it will only connect to the writer.
+func (p GrafeasStorageProvider) ProvideRW(_ string, confi *config.StorageConfiguration) (*storage.Storage, error) {
+	conf, err := rdsconfig.New(confi)
+	if err != nil {
+		return nil, fmt.Errorf("%s, err: %v", errMsgInitConfig, err)
+	}
+
+	// TODO: Use the context passed from main after
+	// the signature of RegisterStorageTypeProvider is updated to include it.
+	writerConnector, err := newConnector(context.Background(), conf, p.drv, p.credentialsCreator, log.Default(), "")
```

Comment:
Do we need to pass `conf.Host` to `newConnector` for writerConnector?

Reply:
conf.Host is the default value. The parameter is used to overwrite the host.